web/frontend/src/components/slam.vue

```diff
@@ -187,41 +212,44 @@ const refreshPCDMap = () => {
               </div>
               <div class="px-2 pt-7">
                 <v-button
-                  v-if="showImage"
+                  v-if="show2d"
                   icon="refresh"
                   label="Refresh"
-                  @click="refreshImageMap()"
+                  @click="refreshImageMap"
```

Comment:
We invoke handlers with `()` elsewhere and did here previously? Any reason why we are removing it? If it works, should we remove the `()` invocations from other handlers as well?

Reply:
Done. Thanks!